<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentpage</class>
 <widget class="QMainWindow" name="studentpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>831</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="collegeTour">
     <widget class="QLabel" name="next_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Campus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="done_button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_button_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QListView" name="select_listView">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="avali_listView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="avali_label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaliable Campuses</string>
      </property>
     </widget>
     <widget class="QLabel" name="select_label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected Campuses</string>
      </property>
     </widget>
     <widget class="QComboBox" name="next_combo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="startCollegeTour">
     <widget class="QLabel" name="start_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Campus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_button_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="avali_label_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Avaliable Campuses</string>
      </property>
     </widget>
     <widget class="QListView" name="start_listView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="start_combo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SortCollegeTour">
     <widget class="QListView" name="sort_listView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>411</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_dist_line">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>430</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="total_dist_label">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>430</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Distance</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
